source-academy/frontend
# Modify mobile keyboard to be scrollable

### Description



Closes #2872 

Modified mobile keyboard key selection. Previously, users had to click a button to cycle through rows of different sets of keys. Now, users scroll horizontally through a single row containing every key.

### Type of change


### How to test


1. Access Source Academy either on a physical mobile device or use browser dev tools to shrink viewport dimensions to access mobile UI.
2. Go to the Source Academy Playground page and expand the draggable keyboard on the right side of the screen.
3. Attempt typing characters into the editor by scrolling to find and clicking the respective keys. Ensure key click is not registered while scrolling. (Note: Editor needs to be in focus (blinking cursor) to be able to type.)

### Checklist
